spec/models/comment_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'Comment model' do
+    before(:each) do
+      first_user = User.create!(name: 'Tom', photo: 'photo.jpg', bio: 'Teacher from Mexico.')
+      first_post = Post.create!(author_id: first_user.id, title: 'Hello', text: 'This is my first post')
+      Comment.create!(post_id: first_post.id, author_id: first_user.id, text: 'Hi Tom!')
+    end
+
+    it 'comment must be present' do
+      expect(Comment.all.length).to eq 2
+    end
```

Comment:
- [ ] This test fails after running `rspec spec`. Please, try to correct it.

<img width="458" alt="image" src="https://user-images.githubusercontent.com/77190203/184426275-22a30e80-eb40-4311-b716-6f6cebf77a7b.png">